includes/settings/js/src/components/fields/TextFields.jsx

```diff
@@ -14,19 +14,48 @@ const TextFields = ({ register, data, editing, fieldId }) => {
 	const fields = models[currentModel]?.fields;
 	const titleFieldId = getTitleFieldId(fields);
 	const showTitleField = !titleFieldId || titleFieldId === fieldId;
-
+	const [showRepeatable, setShowRepeatable] = useState(
+		data?.isRepeatable === true
+	);
 	return (
 		<>
-			{showTitleField && (
+			{data && (
 				<div className="field">
+					<legend>{__("Repeatable Field")}</legend>
```

Comment:
@anthonyburchell These need the "domain" too: 
```suggestion
					<legend>
						{__("Repeatable Field", "atlas-content-modeler")}
					</legend>
```